providers/shared/components/network/id.ftl

```diff
@@ -28,8 +28,46 @@
                 "Children" : [
                     {
                         "Names" : "EnableFlowLogs",
+                        "Description" : "Deprecated: Please use FlowLogs",
                         "Type" : BOOLEAN_TYPE,
                         "Default" : true
+                    },
+                    {
+                        "Names" : "FlowLogs",
+                        "Description" : "Log flows across the network",
+                        "Subobjects" : true,
+                        "Children" : [
+                            {
+                                "Names" : "TrafficType",
```

Comment:
I was expecting this to be TCP, UDP etc. Think type isn't the right word here - maybe "Action"?

Reply:
Only problem with that is the `all` type, not really an action

Reply:
I'll make it any instead of all that will fit action

Reply:
Fixed and ready for review